dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, working at speeds close to RAM limits."
```

Comment:
The `DESCRIPTION` should be wrapped to respect the 80-col limit as in the [Guidelines][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips